### PR TITLE
Add Jest unit tests

### PR DESCRIPTION
**Pull request by kienstra**
#### Changes
* Moves helper functions into `helpers/`, without changing them much
* Adds Jest tests for them

#### Testing
1. npm install
2. npm run test:js
3. Expected: only three dots display
4. npm run dev
5. Expected: the plugin's JS files still work, especially in the block editor

### CODE REVIEW

File: js/src/index.js [hunk 1]
```diff
@@ -1 +0,0 @@
-// Import components.
```
Comment (kienstra) [inline js/src/index.js:1]: It looks like this wasn't used. I could find any reference to it.

File: js/blocks/helpers/registerBlocks.js [hunk 2]
```diff
@@ -9,13 +7,16 @@ import { registerBlockType } from '@wordpress/blocks';
  * Internal dependencies
  */
 import icons from '../../../assets/icons.json';
-import getBlockAttributes from './attributes';
-import { Edit } from '../components';
-import '../../../css/src/editor.scss';
```
Comment (kienstra) [inline js/blocks/helpers/registerBlocks.js:14]: I'm not sure what this was for, it looks like this is enqueued:

https://github.com/getblocklab/block-lab/blob/32e71bbbfbd89802a06ba363976d76e0862119fe/php/blocks/class-loader.php#L154-L159

**Comment by kienstra** [comment]
**Request For Review**

Hi @lukecarbis,
Could you please review this when you have a chance?

**Review by lukecarbis — APPROVED** [review]
Here's my results:

`npm install`
- All good

`npm run test:js`
- Three dots

`npm run dev`
- Everything normal in the plugin

**Comment by kienstra** [comment]
Hi Luke, thanks a lot for testing this!

**Pull request merged**